Appraisals

```diff
@@ -13,13 +13,14 @@ appraise "rails42" do
   gem "railties", "~> 4.2.0"
 end
 
-appraise "rails50" do
-  gem "activerecord", "~> 5.0.0.beta1"
-  gem "railties", "~> 5.0.0.beta1"
-  gem "rspec-rails", github: "rspec/rspec-rails"
-  gem "rspec-support", github: "rspec/rspec-support"
-  gem "rspec-core", github: "rspec/rspec-core"
-  gem "rspec-mocks", github: "rspec/rspec-mocks"
-  gem "rspec-expectations", github: "rspec/rspec-expectations"
-  gem "rspec", github: "rspec/rspec"
+if RUBY_VERSION > "2.2.0"
```

Comment:
I added an [idea on Appraisals](https://github.com/thoughtbot/appraisal/issues/104) to avoid having to do this in the future.